scipy/sparse/linalg/_eigen/lobpcg/lobpcg.py

```diff
@@ -16,12 +16,15 @@
 .. [3] A. V. Knyazev's C and MATLAB implementations:
        https://github.com/lobpcg/blopex
 """
+# Author: Andrew Knyazev <Andrew.Knyazev@ucdenver.edu>
+#         Robert Cimrman <cimrman3@ntc.zcu.cz>
```

Comment:
We keep the history with git and prefer not to have such mentions in the files. It becomes too messy rapidly otherwise.
```suggestion
"""
```
```suggestion
"""
# Author: Andrew Knyazev <Andrew.Knyazev@ucdenver.edu>
#         Robert Cimrman <cimrman3@ntc.zcu.cz>
```

Reply:
done